lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1600,6 +1600,17 @@ func (w *WalletKit) FundPsbt(_ context.Context,
 		account = req.Account
 	}
 
+	var customLockID *wtxmgr.LockID
+	if req.CustomLockId != nil {
```

Comment:
Should do a length check (`len(...) > 0`) instead.

Reply:
Done.